compiler/resolution/lowerIterators.cpp

```diff
@@ -392,7 +392,7 @@ static void markVectorizableForallLoops()
             const char* nom = "SumReduceScanOp";
             size_t len = strlen(nom);
 
-            if (0 == memcmp(nom, opType->symbol->name, len))
+            if (0 == strncmp(nom, opType->symbol->name, len))
```

Comment:
Can this be replaced with startwith? (and remove `len`)

Reply:
I replaced this one and a few other memcmp's that I had put over time.